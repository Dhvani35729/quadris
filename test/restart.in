S
drop
O
right
drop
T
right
right
left
drop
restart
S
clockwise
drop
O
right
clockwise
drop
restart
T
counterclockwise
L
drop
O
drop