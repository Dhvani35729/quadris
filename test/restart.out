Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
J          
JJJ        
           
           
           
           
           
           
           
           
           
           
           
 SS        
SS         
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
           
 SS        
SS         
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 OO        
 OO        
           
           
           
           
           
           
           
           
           
           
           
 SS        
SS         
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
 OO        
 OO        
 SS        
SS         
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
           
           
 OO        
 OO        
 SS        
SS         
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 TTT       
  T        
           
           
           
           
           
           
           
           
           
 OO        
 OO        
 SS        
SS         
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  TTT      
   T       
           
           
           
           
           
           
           
           
           
 OO        
 OO        
 SS        
SS         
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 TTT       
  T        
           
           
           
           
           
           
           
           
           
 OO        
 OO        
 SS        
SS         
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
 TTT       
  T        
 OO        
 OO        
 SS        
SS         
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
S          
SS         
 S         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
  L
LLL


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  L        
LLL        
           
           
           
           
           
           
           
           
           
           
S          
SS         
 S         
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
S          
SS         
 S         
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 OO        
 OO        
           
           
           
           
           
           
           
           
           
           
S          
SS         
 S         
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 OO        
 OO        
           
           
           
           
           
           
           
           
           
           
S          
SS         
 S         
-----------
Next:      
IIII


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
 OO        
SOO        
SS         
 S         
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
T          
TT         
T          
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
  L        
LLL        
           
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
           
           
  L        
LLL        
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
           
  L        
LLL        
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
ZZ         
 ZZ        
           
           
           
           
           
           
           
           
           
           
OO         
OOL        
LLL        
-----------
Next:      
TTT
 T 